Source/cmake/OptionsCommon.cmake

```diff
@@ -296,6 +296,10 @@ elseif (NOT ENABLE_ASSERTS)
     WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-DASSERT_ENABLED=0)
 endif ()
 
+if (ENABLE_DEVELOPER_MODE)
+    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-DENABLE_OFFLINE_ASM_ALT_ENTRY=1)
+endif ()
```

Comment:
What does this actually do? What is the downside to enabling it unconditionally? Performance penalty?

Notably, the stack traces we use to debug bug reports are almost never going to have developer mode enabled. The good news is JavaScriptCore is much more stable than it used to be, so I guess it might not make sense to pay a large performance penalty to get better stack traces. But if the penalty is small, then it's probably worth it.

Reply:
I think I will separate this part into a separate PR, and try to measure the results.